storio-sqlite/src/androidTest/java/com/pushtorefresh/storio/sqlite/impl/QueryTest.java

```diff
@@ -265,23 +290,16 @@
         assertEquals(users.size(), allUsersFromQuery.size());
     }
 
-    private final MapFunc<Cursor, User> mapFuncOnlyEmail = new MapFunc<Cursor, User>() {
-        @NonNull
-        @Override public User map(@NonNull Cursor cursor) {
-            return new User(null, cursor.getString(cursor.getColumnIndex(User.COLUMN_EMAIL)));
-        }
-    };
-
-    @Test public void queryWithRawQuery() {
-
+    @Test
+    public void queryWithRawQuery() {
         final List<User> users = TestFactory.newUsers(20);
 
         int counter = 1;
-        for (User user : users) {
 
+        for (int i = 0; i < users.size(); i++) {
             char[] chars = new char[counter++];
-            Arrays.fill(chars, '*');
-            user.setEmail(new String(chars));
+            Arrays.fill(chars, '*'); // wtf is going on?
```

Comment:
So we need better comment or/and more obvious variable name

2015-04-15 13:07 GMT+03:00 Artem Zinnatullin <notifications@github.com>:

> In
> storio-sqlite/src/androidTest/java/com/pushtorefresh/storio/sqlite/impl/QueryTest.java
> <https://github.com/pushtorefresh/storio/pull/287#discussion_r28407565>:
>
> >              char[] chars = new char[counter++];
> > -            Arrays.fill(chars, '*');
> > -            user.setEmail(new String(chars));
> > +            Arrays.fill(chars, '*'); // wtf is going on?
>
> Haha, no :) I remembered that test but I had to spend couple of minutes to
> understand what's going on
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pushtorefresh/storio/pull/287/files#r28407565>.
>